src/movies.js

```diff
@@ -1,16 +1,99 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let directors = movies.map (function(movie){
+        return movie.director;
+    })
+    return directors;
+} 
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies){
+
+    let occurance = movies.filter(function(movie) {
+        if (movie.director != "Steven Spielberg"){
+            return false;
+        }
+        if (movie.genre.indexOf("Drama")== -1 ) {
+            return false;
+        }
```

Comment:
Because both conditions will return false, you can put them in the same conditional using an or operator ||
Also, using indexOf works good here, but this method is mostly used when you are interested in knowing the index in case it exists. Here you only care if the string is there or not. So using the method .includes would work better as this method automatically returns a boolean :)